docs/hacker-stats-intro/utils.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
```

Comment:
The `utils.py` module can probably be moved to the path `src/bayes_tutorial/utils.py`, which would allow us to use it across all of the notebooks.

Reply:
HBA to do